src/test/java/com/google/api/codegen/testdata/library_pkg.yaml

```diff
@@ -6,6 +6,9 @@ author: Google, Inc.
 email: googleapis-packages@google.com
 homepage: https://github.com/googleapis/googleapis
 license: Apache-2.0
+proto_gen_pkg_deps:
```

Comment:
`proto_gen_pkg_deps` is a bit wordy. How about just `proto_gen_deps` or `proto_deps`